instructors.Rmd

```diff
@@ -0,0 +1,294 @@
+---
+layout: page
+title: Programming with R
+subtitle: Instructor's Guide
+minutes: 0
+---
+
+```{r, include = FALSE}
+source("tools/chunk-options.R")
+```
+
+## Legend
+
+FIXME
+
+## Overall
+
+This lesson is written as an introduction to R, but its real purpose is to
+introduce the single most important idea in programming: how to solve problems
+by building functions, each of which can fit in a programmer's working memory.
```

Comment:
Maybe “building functions” → “decomposing them into smaller pieces”?  Because the decomposition is broader than just functions (and can also include classes, methods, libraries, …).  This text predated this PR though, so I'm fine if we want to punt the tweak into a separate fixup.